### PR TITLE
Bump next-metrics

### PR DESCRIPTION
**Pull request by jkerr321**
to fix failing health check

### CODE REVIEW

File: src/lib/instrument-listen.js [hunk 1]
```diff
@@ -12,14 +12,14 @@ const fs = require('fs');
 const readFile = denodeify(fs.readFile);
 
 module.exports = (app, meta, initPromises) => {
-	async function createServer(app) {
+	async function createServer (app) {
```
Comment (apaleslimghost) [inline src/lib/instrument-listen.js:15]: **nitpick**: there's a lot of unrelated formatting changes, i guess they've come from eslint running on commit? if they have come from eslint, i'd prefer them to be split into a separate PR

**Review by apaleslimghost — APPROVED** [review]

File: src/lib/instrument-listen.js (hunk 1 above)
Comment (jkerr321) [reply src/lib/instrument-listen.js:15]: oo I missed that (clearly not got my head in the game yet today), yes let me undo those....
